controller_interface/test/test_chainable_controller_interface.cpp

```diff
@@ -66,9 +86,13 @@ TEST_F(ChainableControllerInterfaceTest, reference_interfaces_storage_not_correc
   controller.reference_interfaces_.clear();
   auto reference_interfaces = controller.export_reference_interfaces();
   ASSERT_TRUE(reference_interfaces.empty());
+  // expect empty return because storage is not resized
+  controller.internal_state_interfaces_data_.clear();
+  auto internal_state_interfaces = controller.export_internal_state_interfaces();
+  ASSERT_TRUE(internal_state_interfaces.empty());
```

Comment:
```suggestion
  ASSERT_THAT(internal_state_interfaces, IsEmpty());
```
I know I'm nitpicking but let's use the advanced features of gmock. The difference is that you get much more informative prints when something isn't right.

You may need to do `using testing::IsEmpty()` at the top if it isn't there yet